frontend/src/hooks/useDocumentDetail.ts

```diff
@@ -3,18 +3,17 @@ import { fakePromise } from '../helpers';
 import { ApiClient } from '../api/http-common';
 
 export default function useDocumentDetail(id: string) {
-  const client = new ApiClient('http://localhost:3000/'); // note: remove baseUrl argument when api is finally ready
-  const fake = async () => {
-    // get dummy data with latency added
-    const value = await client.get('testdata/document.json', id);
-    return fakePromise(500, value);
-  };
+  const client = new ApiClient(); // note: remove baseUrl argument when api is finally ready
+  // const fake = async () => {
+  //   // get dummy data with latency added
+  //   const value = await client.get('testdata/document.json', id);
+  //   return fakePromise(500, value);
+  // };
 
   return useQuery(
     'document_detail',
-    () =>
-      // client.get(`/documents/${id}`)
-      fake(),
-    { refetchOnWindowFocus: false }
+    () => client.get(`/documents/${id}`, null),
+    // fake(),
+    { refetchOnWindowFocus: false, enabled: id !== undefined }
```

Comment:
Is this kept around for future testing, or are we cleaning out the fake data in a future PR?